formats/licence/publisher/details.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "licence_identifier"
```

Comment:
Depending on the type of licence (as defined by Licensify via the licence identifier), it might:
- redirect to an authority show page with different actions (eg apply, renew)
- show the title of the licence and a postcode search form
- show a radio button for multiple licence authorities

There's an Overview header currently being rendered by Frontend, which we'll update to be conditional based on presence of 'licence_overview'. Publisher itself only validates the presence of the licence identifier.